{"uuid": "6e904c76-0a06-496e-a591-f339df1b34a3", "children": ["82d54f0c-0dc0-46dd-b2ca-7aa300bffc49"], "befores": [{"name": "create_driver", "status": "passed", "start": 1737707811981, "stop": 1737707824812}], "afters": [{"name": "create_driver::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/mohsenakbari/Downloads/Python-Selenium-Pytest-main/tests/conftest.py\", line 72, in create_driver\n    if request.node.rep_call.failed:\n"}, "start": 1737707824841, "stop": 1737707824842}], "start": 1737707811981, "stop": 1737707824842}